en/03_Upgrading/02_Upgrade_to_CMS5.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Upgrading a project to CMS 5
```

Comment:
This probably should just be a general doc about deprecations - this will apply for every major release going forward, so it makes sense for it to stay in a single place throughout the lifetime of Silverstripe CMS

Reply:
I've moved it to the 05_Contributing sections as 18_Deprecations